server/data/fr/gutenberg_12829.txt

```diff
@@ -0,0 +1,140 @@
+Les idées, essences, types, formes, principes, sont éternelles et immuables.
+[Note quatre cent vingt-six : Cette doctrine est partout dans Platon.
+l'Essai sur la Métaphysique d'Aristote, par Monsieur Ravaisson, IIIe part., l.
+deux cent quatre-vingt-onze-trois cent cinq et l'Hist.
+de la philosophie, de Ritter, l.
+deux de la trad., p.
+Il n'y a de parfaitement réel que l'individuel.
+[Note quatre cent vingt-sept : Cat., cinq., Analyt.
+logic., sept, deux cent quarante-six.) [Grec : Ou tina ta koiva.] (Simpl.
```

Comment:
this particular text looks like it has a few problems, like `[` and splitting on abbreviations

Reply:
let's remove it